### PR TITLE
add cone to passenger

### PR DESCRIPTION
**Pull request by Doctor-Cpu**
## About the PR

Adds warning cone to passengers loadout.

## Why / Balance

See many passengers wear it (me included) and feels like it should be added like the gasmask.

## Technical details

N/A

## Media

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

No CL

### CODE REVIEW

**Comment by notquitehadouken** [comment]
i do not think i have ever seen anyone wear a traffic cone ever

**Comment by metalgearsloth** [comment]
Lizards do it a lot.

File: Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml [hunk 1]
```diff
@@ -8,6 +8,19 @@
...
+  equipment: Cone
+  effects:
+  - !type:GroupLoadoutEffect
+    proto: GreyTider
+
+- type: startingGear
+  id: Cone
+  equipment:
+    head: ClothingHeadHatCone
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml:23]: Timegating this one is silly.

**Comment by mirrorcult** [comment]
isnt part of the fun of wearing a cone having to find one in maint

**Comment by Doctor-Cpu** [comment]
> i do not think i have ever seen anyone wear a traffic cone ever

May just be a miros thing.

**Comment by metalgearsloth** [comment]
> isnt part of the fun of wearing a cone having to find one in maint

I'm fine with this as well. I don't think everything should necessarily be a loadout.

File: Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml (hunk 1 above)
Comment (Doctor-Cpu) [reply Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml:23]: Thought it seemed fitting as the others had a time limit and are found in maints.

**Comment by EmoGarbage404** [comment]
I don't think an item that's as generic as this needs to be in loadouts. Some things are fine to be kept as maints loot.

**Pull request closed without merging**